src/components/Timer/timer.js

```diff
@@ -6,20 +6,28 @@ export default class Timer extends Component {
     super(props);
 
     // Initial state with Pomodoro mode, initial Pomodoro duration, and counter
-    this.state = {
-      mode: 'Pomodoro',
-      time: props.timer.pomodoro * 60, // Time in seconds for Pomodoro
-      isRunning: false,
-      pomodoroCount: 0
+    this.state = JSON.parse(localStorage.getItem('timerState'));
```

Comment:
The only problem I see with this is that the timer is saved correctly, but when I go to click play to start it again it doesn't start. It is worth mentioning that I managed to start it again combining a series of buttons. To reproduce this:
- Start the timer.
- Reload the page.
- Play start. (It won't start)
- Play stop, and then start. (It will start)

I don't know what is going on with this, but maybe this steps will help you figure it out.